crates/core_simd/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
```

Comment:
technically we will also need `license` and `description` here if we want to publish to crates.io, but that's no big deal at the moment, and we don't necessarily want that at all anyway. Just that some other parts of the standard library are also available as individual crates (libm, hashbrown, etc), so maybe we could do that if we thought it would make it easier to have people test out intermediate work.

Reply:
I agree--for now I just left it my cargo default.  Everyone else can also add their preferred name/email